tasks/metrics.js

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env node
-
 const execa = require('execa');
```

Comment:
I think we can remove this script and use lint-staged, CQC is way too slow on this task and for injections we have the CI and the deploy process for this.

Reply:
wasnt sure why it is here, but since its run by `node` command, this header is not needed here. (windows dont understand :D)

Reply:
> (windows dont understand :D)

that's the main issue with this OS :trollface: